### PR TITLE
(fix) Wrong symbol param for getPublicTrades request

### PR DESCRIPTION
**Pull request by alexstotsky**
Task: https://app.asana.com/0/1163495710802945/1204298582161856/f
#### Description:
- [x] Fixes issues with incorrect pairs formatting and providing a `symbol` param for the `getPublicTrades` request in some cases
- [x] Adjusts `Symbol Filter` width to fit better for all available pairs
#### Before:
fix_symbol_before

#### After:
fix_symbol_after

### CODE REVIEW

**Review by elvis-poggi — APPROVED** [review]

File: src/state/symbols/utils/index.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ export * from './mapping'
 // BTCUSD -> tBTCUSD
 // BTCF0:USDF0 -> tBTCF0:USDF0
 // USD -> fUSD
-const addPrefix = (symbol = '', isFunding = false) => (
+export const addPrefix = (symbol = '', isFunding = false) => (
```
Comment (ezewer) [inline src/state/symbols/utils/index.js:11]: So i would definitely change this. 
What would do is the following: 
1 - Add t to each result from pairs, that should be the trading pair. 
```console
const tradingPairs = pairs.map(p => `t${p}`)
```

File: src/state/symbols/utils/index.js [hunk 2]
```diff
@@ -61,7 +61,7 @@ export const getSymbolsURL = (symbols) => {
 
 // BTC:USD -> BTCUSD
 // BTCF0:USTF0 -> BTCF0:USTF0
-const deformatPair = pair => ((pair.length === 7) ? pair.replace(':', '') : pair)
+export const deformatPair = pair => ((pair.length === 7) ? pair.replace(':', '') : pair)
```
Comment (ezewer) [inline src/state/symbols/utils/index.js:64]: Same that comment below, there should not be a need to replace ':'

**Review by ezewer — CHANGES_REQUESTED** [review]
The way this was map i dont think its entirely correct. 
Not that this PR introduce the errors, but that the errors where displayed by last changes. 
Logic should be the following: 
1 - From Api we receive pairs, currencies, inactiveSymbols, mapSymbols, inactiveCurrencies
Pairs before being sent to backend should add the t as to get the tranding pair t${pair}
2 - As to display pairs, we should: 
 a - look if pair has its equivalent in mapSymbols
 b - Split the pair into 2 currencies (get each currency from currencies array), this might have ':' as to split or be pair.length === 6 (3 and 3) 
3 - Currencies have an ID and a name to display
4 - In movements display all the symbols 
5 - For rest, filters the one that dont are in trading pairs. 
6 - For the places now added ad `f${symbol.id}` (only the filtered ones)

**Review by ezewer — APPROVED** [review]
Following our recent deliberations, we have decided to proceed with merging this pull request as an interim patch to address the immediate concern. Meanwhile, we will continue working towards a comprehensive solution that effectively resolves all legacy case scenarios.

**Pull request merged**